src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -108,6 +108,41 @@ public void canCreateATitleLevelRequestForOneAvailableCopy()
 
   }
 
+  @Test
+  public void canCreateATitleLevelRequestForOneAvailableCopyWithoutRequestExpirationDate()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    UUID requesterId = usersFixture.jessica().getId();
+    DateTime requestDate = new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+
+    IndividualResource instance = instancesFixture.basedUponDunkirk();
+    IndividualResource holdings = holdingsFixture.defaultWithHoldings(instance.getId());
+
+    final IndividualResource item = itemsFixture.basedUponDunkirkWithCustomHoldingAndLocation(holdings.getId(), null);
+
+    JsonObject requestBody = createInstanceRequestObject(instance.getId(), requesterId,
+      pickupServicePointId, requestDate, null);
+
+    CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+
+    client.post(InterfaceUrls.requestsUrl("/instances"), requestBody,
+      ResponseHandler.any(postCompleted));
+
+    Response postResponse = postCompleted.get(10, TimeUnit.SECONDS);
+
+    JsonObject representation = postResponse.getJson();
+    validateInstanceRequestResponse(representation,
+      pickupServicePointId,
```

Comment:
Should we be validating that `requestExpirationDate` is not set here, as opposed to just validate we don't throw a `NullPointerException`?

Reply:
Yeah, we could do something like that here. I'll put it in the next commit.